docs/rules/jsx-no-target-blank.md

```diff
@@ -20,37 +20,61 @@ This rule aims to prevent user generated links from creating security vulnerabil
 
 * enabled: for enabling the rule. 0=off, 1=warn, 2=error. Defaults to 0.
 * enforce: optional string, 'always' or 'never'
+* Link components can be something other than an `<a>`, see [shared settings](https://github.com/yannickcr/eslint-plugin-react/blob/master/README.md#configuration) for `linkComponents` configuration)
+
+### `enforceDynamicLinks`
+
+#### always
 
-### always (default)
 `{"enforceDynamicLinks": "always"}` enforces the rule if the href is a dynamic link (default)
 
 When {"enforceDynamicLinks": "always"} is set, the following patterns are considered errors:
 
 ```jsx
 var Hello = <a target='_blank' href="http://example.com/"></a>
-var Hello = <a target='_blank' href={ dynamicLink }></a>
+var Hello = <a target='_blank' href={dynamicLink}></a>
 ```
 
 The following patterns are **not** considered errors:
 
 ```jsx
-var Hello = <p target='_blank'></p>
-var Hello = <a target='_blank' rel='noopener noreferrer' href="http://example.com"></a>
-var Hello = <a target='_blank' href="relative/path/in/the/host"></a>
-var Hello = <a target='_blank' href="/absolute/path/in/the/host"></a>
+var Hello = <p target="_blank"></p>
```

Comment:
Was there a specific reason for this change? The rest of the docs use single quotes `'` instead of double quotes `"`. Maybe we should keep things consistent?

Reply:
I made this change; i believe the linter was rrroring on something so i changed them all.

if we want consistency, we should lint our markdown, but also, double quotes is the most common/idiomatic style for jsx attributes in the ecosystem, so I’d prefer we go with that.